tests/OpenSslTest.php

```diff
@@ -30,7 +31,7 @@
  * @license http://www.apache.org/licenses/LICENSE-2.0
  * @link    http://phpsx.org
  */
-class OpenSslTest extends \PHPUnit_Framework_TestCase
```

Comment:
I have updated the PHPUnit version to `^5.6` on master, so we are currently not using `6.0`. Could you please revert those PHPUnit 6.0 changes so that we can update to `6.0` in a separate step? So that we can focus on the `X509` feature.